src/Walker/Collector.C

```diff
@@ -29,69 +29,13 @@ CkReduction::reducerType PDFMerger;
 using walker::Collector;
 
 void
-Collector::chareOrd( const std::vector< tk::real >& ord )
+Collector::chareOrd( const std::vector< tk::real >& ord,
```

Comment:
My point was that it is a bad function name that makes reading the code a lot harder.